src/api/todos.ts

```diff
@@ -0,0 +1,26 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = (userId: number, data: {}) => {
+  return client.post<Todo>(`/todos?userId=${userId}`, data)
+    .then(res => {
+      return {
+        id: res.id,
+        userId: res.userId,
+        title: res.title,
+        completed: res.completed,
+      };
```

Comment:
why not just `return res` ? isn't that the same?

Reply:
oh I see. the response object contain `createdAt` and `updatedAt`, so you can just add this fields to your [Todo](https://github.com/mate-academy/react_todo-app-with-api/pull/587/commits/8c6480855937516674c05013d63c2f045eb58796#diff-0fd93ab189a65a35b743b9b6338535eb73d91cc37074a606eeb45306b290e615R1) type

Reply:
Not resolved.

Reply:
but why do I have to change the type of Todo? Response and Todos have different fields and different aims. Can I instead of it create a new type for Response?

Reply:
> Not resolved.

but why do I have to change the type of Todo? Response and Todos have different fields and different aims. Can I instead of it create a new type for Response?